### PR TITLE
Add Germany as a project

### PR DESCRIPTION
**Pull request by ryanbateman**
README

:wave: I did some awesome work for the Pelias project and would love for everyone to have a look at it and provide feedback.

---
#### Here's the reason for this change :rocket:  
This adds Germany - a whole country, which is good - as a project for use for Pelias geolocation. (Personally speaking, I wanted to setup up a Twitter bot that used Pelias.)  

---
#### Here's what actually got changed :clap:  
I added some basic configuration for German openaddresses data (which, granted, is somewhat sparse) as well as the relevant OpenStreetMap ID. 

---
#### Here's how others can test the changes :eyes:
I did some basic manual tests and queries, though have not added them as docker tests. I can happily add some if required.

### CODE REVIEW

**Review by missinglink — CHANGES_REQUESTED** [review]
Hi, thanks for the project config.

I've added a couple of comments for minor improvements 🎉

File: projects/germany/pelias.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  },
+  "api": {
+    "services": {
+      "pip": { "url": "http://pip:4200" },
+      "libpostal": { "url": "http://libpostal:4400" },
+      "placeholder": { "url": "http://placeholder:4100" },
+      "interpolation": { "url": "http://interpolation:4300" }
+    },
+    "defaultParameters": {
```
[the remark this answers is not in the document]
Comment (ryanbateman) [reply projects/germany/pelias.json]: Interesting! I've now amended these.

File: projects/germany/pelias.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      "focus.point.lat": 51.9,
+      "focus.point.lon": 10.26
+    }
+  },
+  "imports": {
+    "adminLookup": {
+      "enabled": true
```
[the remark this answers is not in the document]
Comment (ryanbateman) [reply projects/germany/pelias.json]: Understood - amended in new commit.

File: projects/germany/pelias.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    "openaddresses": {
+      "datapath": "/data/openaddresses",
+      "files": ["de/berlin.csv", "de/he/city_of_frankfurtammain.csv", "de/he/city_of_kassel.csv", "de/hh/statewide.csv", "de/ni/bentheim.csv", "de/nw/city_of_cologne.csv", "de/nw/city_of_wesel.csv", "de/nw/city_of_wuppertal.csv", "de/nw/statewide.csv", "de/sn/statewide.csv", "de/th/statewide.csv"]
+    },
+    "polyline": {
+      "datapath": "/data/polylines",
+      "files": [ "extract.0sv" ]
+    },
+    "transit": {
```
[the remark this answers is not in the document]
Comment (ryanbateman) [reply projects/germany/pelias.json]: Done!

File: projects/germany/pelias.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      "files": ["de/berlin.csv", "de/he/city_of_frankfurtammain.csv", "de/he/city_of_kassel.csv", "de/hh/statewide.csv", "de/ni/bentheim.csv", "de/nw/city_of_cologne.csv", "de/nw/city_of_wesel.csv", "de/nw/city_of_wuppertal.csv", "de/nw/statewide.csv", "de/sn/statewide.csv", "de/th/statewide.csv"]
+    },
+    "polyline": {
+      "datapath": "/data/polylines",
+      "files": [ "extract.0sv" ]
+    },
+    "whosonfirst": {
+      "datapath": "/data/whosonfirst",
+      "countryCode": "DE",
+      "importPostalcodes": true,
```
Comment (missinglink) [inline projects/germany/pelias.json:55]: Heya, sorry I just double-checked this, the [whosonfirst-data-postalcode-de](https://github.com/whosonfirst-data/whosonfirst-data-postalcode-de) repo which is the source of these postalcodes doesn't actually contain any useful info.

I had a look through the data and all the records are marked non-current and all the lat/lon values are 0,0.

Can we please set `importPostalcodes` to `false`?

Having it as `false` will prevent users downloading that file from our CDN for no reason, saving everyone bandwidth and time 😄

**Review by missinglink — CHANGES_REQUESTED** [review]
Was about to hit merge and found one more minor change 🙏

File: projects/germany/pelias.json (hunk 4 above)
Comment (ryanbateman) [reply projects/germany/pelias.json:55]: Ah, of course - should've spotted that myself. That's a shame - I'll set it to false and resubmit, then see if I can get some updated data there to update it - seems like it should be doable.

Comment (missinglink) [reply projects/germany/pelias.json]: Great thanks, once that's done we can merge :shipit:

**Comment by missinglink** [comment]
Thanks!

**Pull request merged**